### ISSUE TITLE
MenuItem Accelerator can't represent Key Sequences ?

### ISSUE THREAD

**Issue by hexa00** [electron/electron#11740]
From the documentation at: https://electronjs.org/docs/api/accelerator

The accelerator can't be a sequence like if I wanted to do an emacs style keybinding like:

`Ctrl-X Ctrl-F` : Press ctrl-x and then Ctrl-F
 or `Ctrl-X a b`: Press ctrl-x and then a then b

This would be the same for the gmail type keysequence given in example at:
https://github.com/electron/electron/blob/master/docs/tutorial/keyboard-shortcuts.md

```
// gmail style sequences
Mousetrap.bind('g i', () => { console.log('go to inbox') })
Mousetrap.bind('* a', () => { console.log('select all') })
```

Is there a way to have a accelerator for these types of keybindings ?

**Comment by MarshallOfSound** [comment]
> Is there a way to have a accelerator for these types of keybindings ?

Not out of the box but the concept wouldn't be too hard to implement in userland, the issue here is though that any key you register as a global shortcut will not function in other programs.  I.e. Hooking into the `a` will make the `a` key not work everywhere else.  This makes global key sequences make little sense as taking your example code the "g" and "i" key would not work in any other program.

TLDR: Not supported by electron accelerators but you can implement in user land if you want

**Comment by hexa00** [comment]
@MarshallOfSound  Sorry I think I did not express myself properly and the question was miss-understood.

What I mean is that while I can register for example "g" then "i" as a keybinding in userspace without any problems.

I can't have a MenuItem show this as an accelerator, I would like it if the MenuItem would show "g i" but it seems the MenuItem accelerator format only supports 1 chord separated by a + ?
